### PR TITLE
docs: move / add policies that wg-infra is responsible for

### PR DESCRIPTION
**Pull request by MarshallOfSound**
This is 99% moving policies that @electron/wg-infra is now responsible for.  In addition the following changes were made:

* Documented which members of the Infra WG are Slack Owners
* Indicated the NPM access policy is written but not-implemented
* Changed the onboarding documentation to instead just say "use the workflow"
* Added a new `Shared Slack Channels` section in the Slack Access document

TODO after this lands:
* Migrate the offboarding doc into a workflow and change the doc to say "just use the workflow"

### CODE REVIEW

File: wg-infra/policy/joining-governance.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+
+## Requirements
+
+* A new member is approved by their prospective Working Group according to their rules for membership. Approval is recorded in meeting notes.
+* A new member has a GitHub account
+* A new member is a guest on Electron's Slack
+
+## Process
+
+The process for onboarding a new member into Electron Governance is documented and governed via a workflow in the #wg-infra channel.  To use it please ensure the above requirements are met and then trigger the `Onboard Maintainer` workflow.  This workflow will guide you through the process and ping the appropriate teams when required.
```
Comment (codebytere) [inline wg-infra/policy/joining-governance.md:13]:
```suggestion
The process for onboarding a new member into Electron Governance is documented and governed via a workflow in the #wg-infra channel.  To use it please ensure the above requirements are met and then trigger the `Onboard Maintainer` workflow. This workflow will guide the onboarding process and ping the appropriate teams when required.
```

i’d also heavily prefer that even if we’re delegating to the onboarding workflow, we still enumerate the steps the bot takes for reference and transparency.

**Review by codebytere — COMMENTED** [review]
Mostly changes suggested to keep the documentation in consistent perspective - otherwise LGTM

File: wg-infra/policy/access/npm.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+
+There are three teams on NPM, `developers`, `cfa`, `electron`.
+
+* `developers` will have `read` access to all packages with the exception of the "electron" package.
+* `cfa` will have `read/write` on all packages with the exception of the "electron" package.
+* `electron` will have `read/write` on **only** the "electron" package.
+
+The only user in the `electron` team will be the "electron-bot" user.  As such the only user with permission to publish the `electron` package should always be "electron-bot".  Publishing of this package will be triggered through `sudowoodo`.
+
+The only user in the `cfa` team will be the "electron-cfa" user.  As such the only user with permission to publish packages in the `electron` organization should be "electron-cfa".  As no humans have publish rights to any of these packages they should all be configured with `semantic-release` and the `@electron/semantic-release-npm-cfa` plugin.  For information on how to configure this plugin for use with a new package head over to [`electron/cfa`](https://github.com/electron/cfa).
```
Comment (dsanders11) [inline wg-infra/policy/access/npm.md:19]: Is `electron/cfa` still the correct repo, despite being archived?

**Comment by dsanders11** [comment]
General comment, the relative links inside Markdown files in `wg-infra/policy` and `wg-infra/policy/access` are broken, they need to go back up more levels.

**Review by dsanders11 — APPROVED** [review]

**Review by codebytere — APPROVED** [review]

**Pull request merged**